samples/polyfills/buffer/azion/readme.md

```diff
@@ -0,0 +1,19 @@
+# Buffer through polyfills
+
+This code sample is JavaScript project that demonstrates the use of the Buffer class from the Node.js buffer module inside an **edge function** through the use of polyfills. 
+
+## How it Works
+
+The main function in this project is myWorker(event). Here's a brief rundown of what it does:
+
+1. It creates two Buffer instances, buf1 and buf2, both initialized with the string "x".
+
+2. It then compares these two buffers using the Buffer.compare method. This method returns a number indicating whether buf1 is equal to buf2. In this case, it'll return 0, because they're equal.
```

Comment:
```suggestion
2. It then compares these two buffers using the `Buffer.compare` method. This method returns a number indicating whether `buf1` is equal to `buf2`. In this case, it'll return `0`, because they're equal.
```